specification/postgresqlhsc/resource-manager/readme.python.md

```diff
@@ -0,0 +1,21 @@
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+Please also specify `--python-sdks-folder=<path to the root directory of your azure-sdk-for-python clone>`.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
+  namespace: Microsoft.DBforPostgreSQL
```

Comment:
Namespace should be change to avoid conflict.

Reply:
namespace for python should like "azure.mgmt.xxx". Also, output folder is not right in 'readme.python.md'.  please reference https://github.com/Azure/azure-rest-api-specs/blob/master/specification/datadog/resource-manager/readme.python.md

Reply:
I updated namespace and structure of python readme from the reference.